swagger/api/generic/severity.yaml

```diff
@@ -0,0 +1,9 @@
+severity:
+  title: Severity
+  description: This is the over all severity or the severity of the most significant item.
+  enum:
+    - indetermined
+    - info
```

Comment:
In other places I see uppercase ENUMs, which are my preference. As long as we make them consistent.